### ISSUE TITLE
License type

### ISSUE THREAD

**Issue by deining** [ralphmlrp/HitRestClient#2]
Thanks for providing this useful project.
Can you please specify he type of license associated with this project?
This is important for anyone considering the use of this code inside his own project(s), it maybe open or proprietary code.

**Comment by NiklasDah** [comment]
Yes. The license isn’t noted here, currently - in the documentation it states „freely available“, though:
https://www.hi-tier.de/HitCom3/apiHelp

I don’t think, they are responding to GitHub issues, tho. This repository should generally be moved away from a private GitHub Account.
